load_site.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import aiohttp
+import time
+
+
+async def load_yandex():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://ya.ru')
+        print(await resp.json())
+
+
+async def load_google():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://google.com')
+        print(await resp.json())
+
+
+async def main():
+    while True:
+        await load_yandex()
+        await load_google()
+
+        time.sleep(1)
```

Comment:
use async sleep
```suggestion
        asyncio.sleep(1)
```

Better move .sleep() call to the task function body.

Value should be moved to a constant.